tasks/build_tags.py

```diff
@@ -4,8 +4,10 @@
 import sys
 from invoke import task
 
-# ALL_TAGS lists any available build tag
+# ALL_TAGS lists all available build tags.
+# Used to remove unknown tags from provided tag lists.
 ALL_TAGS = set([
```

Comment:
This should either be a list or instead everything should be a set. I like everything being a set but I don't know if there is something that prevents it.

Reply:
I converted every static list to a set. It sadly doesn't simplify the `get_build_tags` function, as the `include` and `exclude` parameters can be lists.